lte/gateway/c/session_manager/SessionState.h

```diff
@@ -82,6 +82,7 @@ class SessionState {
     std::vector<std::string> static_rules;
     std::vector<PolicyRule> dynamic_rules;
     std::vector<PolicyRule> gy_dynamic_rules;
+    std::vector<std::string> restrict_rules;
```

Comment:
Do you think instead of keeping track of both `gy_dynamic_rules` and `restrict_rules`, do you think we can just refer to the FinalUnitACtion stored? It looks like we can always re-construct redirect dynamic rule and the set of restrict_rules from that struct. What do you think?

Reply:
for my education: are `gy_dynamic_rules` the ones used on redirection only? or are those rules used somewhere else?

Reply:
As discussed, I removed the restrict rules from session state for now. We will see if we can do without when retrieving gy statistics from pipelined.